Main.java

```diff
@@ -9,7 +11,34 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+
+   //ask user for an integers
+   System.out.println("Enter an integer");
+
+    //get integer from user
+  int integer1 = input.nextInt();
+  
+   //ask user to chose another integer
+  System.out.println("Enter another integer");
+
+  //get integer from user
+  int integer2 = input.nextInt();
+
+  //calculate greater number
+  if(integer1 < integer2){
+  
+  //tell user their numbers in ascending order
+  System.out.println("Your numbers in ascending order are " + integer1 + "," + integer2);
+
+ } else if (integer2 > integer1){
+
+  //tell user their number in ascending order
+  System.out.println("Your numbers in acending order are " + integer1 + "," +  integer2);
+  
```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement.

Otherwise, this is good looking code!